lib/adal.js

```diff
@@ -684,6 +720,10 @@ AuthenticationContext.prototype.saveTokenFromHash = function (requestInfo) {
                 // save token with related resource
                 this._saveItem(this.CONSTANTS.STORAGE.ACCESS_TOKEN_KEY + resource, requestInfo.parameters[this.CONSTANTS.ACCESS_TOKEN]);
                 this._saveItem(this.CONSTANTS.STORAGE.EXPIRATION_KEY + resource, this._expiresIn(requestInfo.parameters[this.CONSTANTS.EXPIRES_IN]));
+                if (window.parent && window.parent.loadframeStatus) {
```

Comment:
We need this check when aad returns an error in the if block above line #700. Since, we received the response, even if its error, we should delete the loadframestatus so that its no longer pending.